python/tvm/relay/op/transform.py

```diff
@@ -1450,6 +1450,61 @@ def sparse_reshape(sparse_indices, prev_shape, new_shape):
     return TupleWrapper(_make.sparse_reshape(sparse_indices, prev_shape, new_shape), 2)
 
 
+def segment_sum(data, indices, num_segments=None):
+    """
+    Computes the sum along segments of a tensor. This op is much better understood with
+    visualization articulated in the following links and examples at the end of this docstring.
+
+    https://www.tensorflow.org/api_docs/python/tf/raw_ops/UnsortedSegmentSum?hl=fr
+    https://caffe2.ai/docs/sparse-operations.html#null__unsorted-segment-reduction-ops
+
+    Parameters
+    ----------
+    data : relay.Expr
```

Comment:
Is this tensor 1D? If not, is the reduction over all dimensions? Or just one?

Reply:
No, . The reduction is only over the zeroth dimension. I clarified it in the documentation one.